xbmc/input/ButtonTranslator.cpp

```diff
@@ -1155,9 +1155,8 @@ int CButtonTranslator::GetFallbackWindow(int windowID)
     if (fallbackWindows[index].origin == windowID)
       return fallbackWindows[index].target;
   }
-  // for addon windows use WINDOW_ADDON_START
-  // because id is dynamic
-  if (windowID >= WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
+  // for addon windows use WINDOW_ADDON_START because id is dynamic
+  if (windowID > WINDOW_ADDON_START && windowID <= WINDOW_ADDON_END)
```

Comment:
BTW we use the same fallback handling all over the place

https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L1174
https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L1068
https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L1045

and the one I've added in this PR.